src/components/AddRestaurantModal/AddRestaurantModal.jsx

```diff
@@ -0,0 +1,52 @@
+import "./AddRestaurantModal.css";
+
+function AddRestaurantModal() {
+  return (
+    <div className="modal modal--open">
+      <div className="modal-backdrop"></div>
```

Comment:
```suggestion
      <div className="modal-backdrop"/>
```
HTML이랑 다르게 JSX는 모든 태그가 `self closing`이 가능해서 내부 요소가 없다면 `self closing` 해줘도 좋을 것 같아요🙃

Reply:
self closing 개념에 대해 잘 알게 되었습니다 감사합니당 🙃